### ISSUE TITLE
How to backpropagate the gradients of the embeddings output by the image processor to the input image tensor?

### ISSUE THREAD

**Issue by weiminbai** [QwenLM/Qwen2.5-VL#1168]
I'm using the processor of Qwen2.5-VL, and the image processor within it should be Qwen2ImageProcessor. The input image I provide is a PyTorch tensor with gradients, and the processor outputs the feature embeddings of the image. How can I ensure that the gradient flow is not interrupted during this process?

**Comment by JJJYmmm** [comment]
same as #1167
